src/UpcomingEvents.js

```diff
@@ -12,13 +12,20 @@ function UpcomingEvents() {
 
   return (
     <div className="event-card__wrapper">
-      {upcomingEvents.map(function(obj, i) {
-        return (
-          <a href={`/events/${obj.path}`} className="event-card__wrapper--link">
-            <EventCard key={i} row={obj} />
-          </a>
-        );
-      })}
+      {upcomingEvents.length === 0 ? (
+        <h3 class="no-event-message">You have no upcoming events</h3>
```

Comment:
Since the events are general, I think "There are no upcoming events" makes sense. We may also want to make this customizable via a module field.